examples/guided.py

```diff
@@ -97,33 +121,45 @@ def ask_user_questions():
 
 					# Get a valid & supported filesystem for the partition:
 					while 1:
-						new_filesystem = input(f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): ").strip(' ')
+						new_filesystem = input(
+							f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): "
+						).strip(' ')
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This partially decreases readability.